.github/workflows/ci.yml

```diff
@@ -71,15 +72,50 @@ jobs:
           files_ignore: |
             **/.gitignore
             **/*.md
+            workspace-contracts/**
           json: true
 
+      - name: Look for changes in workspace contracts
+        id: CHANGED_WORKSPACE_CONTRACTS
+        uses: tj-actions/changed-files@v39
+        with:
+          dir_names: "true"
+          dir_names_exclude_current_dir: "true"
+          dir_names_max_depth: 2
+          files: |
+            workspace-contracts/**
+          files_ignore: |
+            **/.gitignore
+            **/*.md
+            multi-contract-caller/**
+            upgradeable-contracts/**
```

Comment:
These are redundant?

Reply:
Oh maybe, yes, haven't pay much attention to changed-files macro, but looks redundant, yes.

Reply:
Please remove then we can merge

Reply:
done! thanks!